src/commissaire_http/handlers/clusters.py

```diff
@@ -25,11 +29,222 @@ def list_clusters(message, bus):
 
     :param message: jsonrpc message structure.
     :type message: dict
+    :param bus: Bus instance.
+    :type bus: commissaire_http.bus.Bus
+    :returns: A jsonrpc structure.
+    :rtype: dict
+    """
+    clusters_msg = bus.request('storage.list', params=['Clusters'])
+    return create_response(
+        message['id'],
+        [cluster['name'] for cluster in clusters_msg['result']])
+
+
+def get_cluster(message, bus):
+    """
+    Lists all clusters.
+
+    :param message: jsonrpc message structure.
+    :type message: dict
+    :returns: A jsonrpc structure.
+    :rtype: dict
+    """
+    cluster = bus.request(
+        'storage.get', 'get', params=[
+            'Cluster', {'name': message['params']['name']}])
+    return create_response(message['id'], cluster['result'])
```

Comment:
The old "get cluster" logic did some on-the-fly calculations here for total/available/unavailable hosts, to supplement what's stored in etcd.  If you still want that, I think it would make more sense for that to live here than in the storage service.  (Or at least add a TODO comment.)